File: CRM2/Models/Med.cs
using CRM2.Models.BaseEntityModel;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace CRM2.Models
{
    public class Med : BaseEntity
    {
        //[Key]
       // public int med_id { get; set; }
        [MaxLength(200)]
        public String med_name { get; set; }
        [MaxLength(500)]
        public String med_description { get; set; }
        [MaxLength(1500)]
        public String med_dosage_freq { get; set; }
        public DateTime med_start_date { get; set; }
        public DateTime med_end_date { get; set; }
        [MaxLength(500)]
        public String med_category { get; set; }
        [ForeignKey("Visit")]
        public int visit_id { get; set; }
        public Visit Visit { get; set; }





    }
}

File: CRM2/ClientApp/src/app/_model/Med.ts
export class Med {
  constructor(
    public Id: number,
    public CreatedDate: string,
    public med_name: string,
    public med_description: string,
    public med_dosage_freq: string,
    public med_start_date: string,
    public med_end_date: string,
    public med_category: string,
    public visit_id: number
  )
  {
  }
}
